### ISSUE TITLE
Please add a Dockerfile or upload it to Docker Hub.

### ISSUE THREAD

**Issue by mwnu** [Cyberes/MatrixGPT#2]
Docker installation is a simpler method, and future updates are also easier.

**Comment by Cyberes** [comment]
Good idea, but I have my hands full with a different project right now. I'll get around to it eventually.
